typespec-extension/changelog.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 0.12.2 (2024-01-09)
+
+Compatible with compiler 0.51.
+
+- Supported convenience API for "application/merge-patch+json"(basic functionality).
```

Comment:
nit, space

```suggestion
- Supported convenience API for "application/merge-patch+json" (basic functionality).
```

Reply:
Should we add that is only available under stream-style?

Reply:
updated